xgettext.js

```diff
@@ -15,6 +15,7 @@ var XGettext = module.exports = function( options ) {
 	this.options = _.extend( {}, XGettext.defaultOptions, options );
 	this.options.keywords = this._normalizeKeywords( this.options.keywords );
 	this.options.keywordFunctions = Object.keys( this.options.keywords );
+	this.options.parserOptions = _.extend( {}, options.parserOptions );
```

Comment:
Probably belongs in a separate PR, but if a user passes in `{locations:false}` we crash trying to pull properties out of `loc`.  We should probably rename it to something like 'requiredOptions', and add it in as the last argument to `_.extend()` on line 15 so it overwrites the user options.